src/btrfs.c

```diff
@@ -79,23 +131,23 @@ static void btrfs_submit_value(const char *folder, const char *error,
 
   // create value
   sstrncpy(vl.plugin, PLUGIN_NAME, sizeof(vl.plugin));
-  snprintf(vl.plugin_instance, sizeof(vl.plugin_instance), "%s#%s", folder,
-           error);
+  snprintf(vl.plugin_instance, sizeof(vl.plugin_instance), "%s", folder);
+  snprintf(vl.type_instance, sizeof(vl.type_instance), "%s", error);
   sstrncpy(vl.type, "count", sizeof(vl.type));
```

Comment:
You use `count` type here. This type is GAUGE, but nobody is interested in absolute value of errors on charts.
For these metrics, sysadm is interested in error rate, and there is DERIVE types intended for that.
If you want to use existing type, I recommend use of `errors` type:

```
errors                  value:DERIVE:0:U
```

Reply:
Ah okay, it was not clear to me that GAUGE is for charts. I change this :+1:

Reply:
All collectd metrics are mainly for charts )
Hope, your task is solved too!